pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/v3/AdminApiTransactionMultiBrokerTest.java

```diff
@@ -0,0 +1,107 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pulsar.broker.admin.v3;
+
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.pulsar.broker.transaction.TransactionTestBase;
+import org.apache.pulsar.client.api.Consumer;
+import org.apache.pulsar.client.api.Message;
+import org.apache.pulsar.client.api.PulsarClient;
+import org.apache.pulsar.client.api.Schema;
+import org.apache.pulsar.client.api.transaction.Transaction;
+import org.apache.pulsar.common.naming.TopicName;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+@Slf4j
+public class AdminApiTransactionMultiBrokerTest extends TransactionTestBase {
+
+    private static final int NUM_BROKERS = 16;
+    private static final int NUM_PARTITIONS = 16;
+    
+    @BeforeMethod
+    protected void setup() throws Exception {
+        setUpBase(NUM_BROKERS, NUM_PARTITIONS, NAMESPACE1 + "/test", 0);
```

Comment:
Maybe we need to set the configuration `allowAutoTopicCreation` to `false`, according to the description in the issue https://github.com/apache/pulsar/issues/14790 the `allowAutoTopicCreation` is `false`.

Reply:
I think that was used to test another question. allowAutoTopicCreation is false, has nothing to do with this PR, and we use admin in setupBase to create the required topic.

Reply:
Does this PR is used to fix issue #14790? If so, maybe we need to create the same environment for the issue.

Reply:
When we don't know what the problem is, we need to set up the same environment to reproduce the problem. And when we fix the problem, we only need to focus on the problem itself

Reply:
I can start a cluster of brokers with the same configuration as the issue to test this modification. If there is no problem, I don't think we need to include extraneous variables in CI

Reply:
And this is not fix the issue https://github.com/apache/pulsar/issues/14790

Reply:
OK. It seems that this PR is linked with issue #14790 before.

Reply:
My fault, did not read this issue carefully